view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -139,7 +139,7 @@ define(
              * @return {integer}
              */
             getTotal: function () {
-                return + window.checkoutConfig.totalsData.grand_total;
+                return parseInt(window.checkoutConfig.quoteData.grand_total);
```

Comment:
A few things:

a) Is `totalsData.grand_total` a total in the correct currency?
b) Is it definitely an integer? Could `parseInt` break stuff here
c) The original code was already ensuring it was ensuring a number was returned - the unary plus does this conversion (coercion) for you

Reply:
a) Yes, This returns correct currency.
b) It is an Integer, but stored as string ("300.0000")
c) will update.